doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -195,6 +208,282 @@ Response:
 
 [Top](#top)
 
+###<a name="section5.2"></a>`POST /v1/namemappings`
+Adds new [name mappings](../../cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md) to the configured ones. The new mappings are given within the request payload, following the Name Mappings Json format (for instance, if a new entity mapping has to be added, the service and the service path it belongs to must be present in the Json as well). All the mappings within the payload that are not present in the configuration are added.
```

Comment:
Wherever you mention "Name Mapping Json format" a reference to the place in the documentation where that format is described would be great (not sure, but that place seems to be ngsi_name_mappings_interceptor.md#section1.1).

Reply:
Fix in 152130135685df05a57075847dee7166922c91a6